lifting-state-up/lifting-state-up.md

```diff
@@ -201,22 +184,23 @@ Now is a great time to read through our code and delete any unused imports. In p
 <!-- prettier-ignore-start -->
 ### !challenge
 * type: multiple-choice
-* id: 8ffcf3b5
+* id: 42c4e728
```

Comment:
Why do you need to generate a new id? Is it because `8ff` is associated with the question as it previously appeared?

Reply:
That's why I did it, yes. Since the problem is effectively a new question, I figured a new id (which would sever the connection to any previous answer) was called for. Probably not that important, since we don't back-publish to previous cohorts,but 🤷